pkg/build/generator/generator.go

```diff
@@ -218,6 +218,10 @@ func (g *BuildGenerator) Instantiate(ctx kapi.Context, request *buildapi.BuildRe
 		return nil, err
 	}
 
+	if bc.Spec.Strategy.ExternalStrategy != nil {
+		return nil, &GeneratorFatalError{fmt.Sprintf("can't instantiate from BuildConfig %s/%s: BuildConfig uses an External build strategy", bc.Namespace, bc.Name)}
+	}
```

Comment:
i think we ultimately are going to want to instantiate these, just like any other BC, we'll instantiate it to create a build object, and then something will be monitoring for those new build jobs to bridge into launching the jenkins job.